writer.go

```diff
@@ -0,0 +1,145 @@
+package pinecone
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/pinecone-io/go-pinecone/pinecone"
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+// Writer implements a writer logic for Sap hana destination.
+type Writer struct {
+	client *pinecone.Client
+	index  *pinecone.IndexConnection
+}
+
+// NewWriter New creates new instance of the Writer.
+func NewWriter(ctx context.Context, config DestinationConfig) (*Writer, error) {
+	sdk.Logger(ctx).Trace().Msg("Creating new writer.")
+
+	sdk.Logger(ctx).Error().Msgf("API: %v, INDEX:%v", config.PineconeAPIKey, config.PineconeHostURL)
```

Comment:
It's probably better not to log the API key, so we don't expose sensitive information in the logs.

Reply:
Refactored most logging calls in later prs.